<ui version="4.0" >
 <author>Kovid Goyal</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>822</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>libprs500</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >:/library</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="LocationView" name="location_view" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>10000</width>
          <height>90</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy" >
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy" >
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="dragDropMode" >
         <enum>QAbstractItemView::DragDrop</enum>
        </property>
        <property name="flow" >
         <enum>QListView::TopToBottom</enum>
        </property>
        <property name="spacing" >
         <number>20</number>
        </property>
        <property name="viewMode" >
         <enum>QListView::IconMode</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vanity" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="text" >
         <string>For help visit &lt;a href="https://libprs500.kovidgoyal.net/wiki/GuiUsage">http://libprs500.kovidgoyal.net&lt;/a>&lt;br>&lt;br>&lt;b>libprs500&lt;/b>: %1 by &lt;b>Kovid Goyal&lt;/b> &amp;copy; 2007&lt;br>%2</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::RichText</enum>
        </property>
        <property name="openExternalLinks" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" >
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&amp;Search:</string>
        </property>
        <property name="buddy" >
         <cstring>search</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SearchBox" name="search" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="toolTip" >
         <string>Search the list of books by title or author&lt;br>&lt;br>Words separated by spaces are ANDed</string>
        </property>
        <property name="whatsThis" >
         <string>Search the list of books by title, author, publisher, tags and comments&lt;br>&lt;br>Words separated by spaces are ANDed</string>
        </property>
        <property name="autoFillBackground" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="frame" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="clear_button" >
        <property name="toolTip" >
         <string>Reset Quick Search</string>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="images.qrc" >:/images/clear_left.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" >
     <widget class="QStackedWidget" name="stack" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>100</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex" >
       <number>2</number>
      </property>
      <widget class="QWidget" name="library" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="BooksView" name="library_view" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>100</horstretch>
            <verstretch>10</verstretch>
           </sizepolicy>
          </property>
          <property name="acceptDrops" >
           <bool>true</bool>
          </property>
          <property name="dragEnabled" >
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode" >
           <bool>false</bool>
          </property>
          <property name="dragDropMode" >
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="main_memory" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="DeviceBooksView" name="memory_view" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>100</horstretch>
            <verstretch>10</verstretch>
           </sizepolicy>
          </property>
          <property name="acceptDrops" >
           <bool>true</bool>
          </property>
          <property name="dragEnabled" >
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode" >
           <bool>false</bool>
          </property>
          <property name="dragDropMode" >
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="DeviceBooksView" name="card_view" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>100</horstretch>
            <verstretch>10</verstretch>
           </sizepolicy>
          </property>
          <property name="acceptDrops" >
           <bool>true</bool>
          </property>
          <property name="dragEnabled" >
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode" >
           <bool>false</bool>
          </property>
          <property name="dragDropMode" >
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="tool_bar" >
   <property name="minimumSize" >
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="action_add" />
   <addaction name="action_del" />
   <addaction name="action_save" />
  </widget>
  <widget class="QStatusBar" name="statusBar" >
   <property name="mouseTracking" >
    <bool>true</bool>
   </property>
  </widget>
  <action name="action_add" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/plus.svg</iconset>
   </property>
   <property name="text" >
    <string>Add books</string>
   </property>
   <property name="shortcut" >
    <string>A</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
  </action>
  <action name="action_del" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/trash.svg</iconset>
   </property>
   <property name="text" >
    <string>Remove books</string>
   </property>
   <property name="toolTip" >
    <string>Remove books</string>
   </property>
   <property name="shortcut" >
    <string>Del</string>
   </property>
  </action>
  <action name="action_edit" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/edit_input.svg</iconset>
   </property>
   <property name="text" >
    <string>Edit meta information</string>
   </property>
   <property name="shortcut" >
    <string>E</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
  </action>
  <action name="action_sync" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/sync.svg</iconset>
   </property>
   <property name="text" >
    <string>Send to device</string>
   </property>
  </action>
  <action name="action_save" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/save.svg</iconset>
   </property>
   <property name="text" >
    <string>Save to disk</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BooksView</class>
   <extends>QTableView</extends>
   <header>library.h</header>
  </customwidget>
  <customwidget>
   <class>SearchBox</class>
   <extends>QLineEdit</extends>
   <header>library.h</header>
  </customwidget>
  <customwidget>
   <class>LocationView</class>
   <extends>QListView</extends>
   <header>widgets.h</header>
  </customwidget>
  <customwidget>
   <class>DeviceBooksView</class>
   <extends>QTableView</extends>
   <header>library.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>search</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>853</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel" >
     <x>784</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
